### PR TITLE
Get rid of the unwanted tooltip copying (final)

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

Follow-up to https://github.com/home-assistant/frontend/pull/7408.

This new approach now works in both Firefox and Chrome.

The "card-content" is selected and then a temporary heading added (fixed in EN) that is removed again after the copy execution.

## Type of change

- [ ] Dependency upgrade
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/info/system-health-card.ts [hunk 1]
```diff
@@ -128,30 +128,28 @@ class SystemHealthCard extends LitElement {
...
+    const copyElement = this.shadowRoot?.querySelector(
       ".card-content"
     ) as HTMLElement;
 
-    range = document.createRange();
+    // Add temporary heading (fixed in EN since usually executed to provide support data)
+    copyElement.insertAdjacentHTML(
+      "afterbegin",
+      '<h3 id="temporary">System Health</h3>'
+    );
```
Comment (bramkragten) [inline src/panels/config/info/system-health-card.ts:139]:
```suggestion
	const tempTitle = document.createElement("h3");
	tempTitle.innerText = "System Health";
	copyElement.prepend(tempTitle);
	copyElement.removeChild(tempTitle);
```

Comment (bramkragten) [reply src/panels/config/info/system-health-card.ts:139]: Or use `insertBefore` as I'm not sure `prepend` is supported in all browsers

**Comment by bramkragten** [comment]
Thinking now, do we really need the header? Would it not be enough to just copy the content?

**Comment by spacegaier** [comment]
> 
> 
> Thinking now, do we really need the header? Would it not be enough to just copy the content?

I had thought about that as well, but I think it makes sense to have it, since we also have the "Lovelace" title in the middle, so it is consistent.

File: src/panels/config/info/system-health-card.ts (hunk 1 above)
Comment (spacegaier) [reply src/panels/config/info/system-health-card.ts]: Yes `prepend` is still experimental, so I went with `insertBefore`.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**